File: Unidad3/Java/U3p4.java
package com.mycompany.Programa4_U3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class u3p4 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int dia;
        System.out.println("Programa para los meses del año.");
        System.out.println("Introdusca un número para saber que mes es: ");
        dia=sc.nextInt();
                switch(dia){
                    case 1:
                        System.out.println("Es enero");
                        break;
                    case 2:
                        System.out.println("Es febrero");
                        break;
                    case 3:
                        System.out.println("Es marzo");
                        break;
                    case 4:
                        System.out.println("Es abril");
                        break;
                    case 5:
                        System.out.println("Es mayo");
                        break;
                    case 6:
                        System.out.println("Es junio");
                        break;
                    case 7:
                        System.out.println("Es julio");
                        break;
                    case 8:
                        System.out.println("Es agosto");
                        break;
                    case 9:
                        System.out.println("Es sepetiembre");
                        break;
                    case 10:
                        System.out.println("Es octubre");
                        break;
                    case 11:
                        System.out.println("Es noviembre");
                        break;
                    case 12:
                        System.out.println("Es diciembre");
                        break;
                }
    }
}

File: Unidad3/C#/u3p4.cs
using System;

namespace u3p4
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int dia;
            Console.WriteLine("Programa para los meses del año.");
            Console.WriteLine("Introdusca un número para saber que mes es: ");
            dia = Convert.ToInt32(Console.ReadLine());
            switch (dia)
            {
                case 1:
                    Console.WriteLine("Es enero");
                    break;
                case 2:
                    Console.WriteLine("Es febrero");
                    break;
                case 3:
                    Console.WriteLine("Es marzo");
                    break;
                case 4:
                    Console.WriteLine("Es abril");
                    break;
                case 5:
                    Console.WriteLine("Es mayo");
                    break;
                case 6:
                    Console.WriteLine("Es junio");
                    break;
                case 7:
                    Console.WriteLine("Es julio");
                    break;
                case 8:
                    Console.WriteLine("Es agosto");
                    break;
                case 9:
                    Console.WriteLine("Es sepetiembre");
                    break;
                case 10:
                    Console.WriteLine("Es octubre");
                    break;
                case 11:
                    Console.WriteLine("Es noviembre");
                    break;
                case 12:
                    Console.WriteLine("Es diciembre");
                    break;
            }
        }
    }
}
